px-deck-selector.html

```diff
@@ -122,98 +121,79 @@
       }
     },
 
-    _setDecks: function() {
-      if (this._isDecks && (this.selectedDeck == null || this.selectedDeck.constructor === Object)) {
-        this.set('selectedDeck', this.decks[0]);
-      }
+    created: function() {
+      this._handleDocumentTapped = this._handleDocumentTapped.bind(this);
     },
 
-    _computeIsDecks: function(decks){
-      return (decks && decks.length > 0);
+    /**
+     * Always selects the first deck when `decks` are defined.
+     */
+    _selectFirstDeck: function(decks) {
+      if (Array.isArray(decks) && decks.length > 0) {
+        this.selectedDeck = decks[0];
+      }
     },
 
-    _hideChevronIfNoDecks: function(isDecks){
-      return isDecks ? '' : 'visuallyhidden';
+    _handleDropdownOpenedChanged: function(opened) {
+      if (opened) {
+        document.addEventListener('click', this._handleDocumentClicked, false);
```

Comment:
tap instead of click?

Reply:
I don't think there is a native `tap` event in the browser, i think `on-tap` is just Polymer sugar. I could be wrong though.